CHANGELOG.md

```diff
@@ -1,6 +1,15 @@
 # CHANGELOG
 
 
+## vN.N.N
```

Comment:
@jalopez13 - please fix CHANGELOG, both `vN.N.N` and `patch` sections are in here

Reply:
@arunkarnati 

Updated changelog... Not sure how it should be formatted.